src/index.html

```diff
@@ -5,8 +5,477 @@
     <meta name="viewport" content="width=device-width, initial-scale=1.0">
     <title>Catalog</title>
     <link rel="stylesheet" href="./styles/main.scss">
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+    <link rel="preconnect" href="https://fonts.gstatic.com">
+    <link href="https://fonts.googleapis.com/css2?family=Roboto:wght@400;500;700&display=swap" rel="stylesheet">
   </head>
   <body>
-    <h1>Catalog</h1>
+    <header class="header">
+      <div class="logo">
+        <a href="#" class="logo__link">
+          <img src="./images/logo.png" alt="logo-img">
+        </a>
+      </div>
+    
+      <nav class="nav">
+        <ul class="nav-list">
+          <li class="nav-value">
+            <a class="is-active" href="#Apple">
+              apple
```

Comment:
```suggestion
              Apple
```
it's better to uppercase first letter